app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -0,0 +1,24 @@
+.gem-c-intervention {
+  @include govuk-text-colour;
+  @include govuk-responsive-padding(4);
+  @include govuk-responsive-margin(4, "bottom");
+
```

Comment:
Nitpick: is this empty line needed?

Reply:
nope. I just picked the wrong component to copy from ;)